engine/execution/computation/computer/computer.go

```diff
@@ -155,20 +155,38 @@ func (e *blockComputer) executeBlock(
 		},
 	}
 
+	eh := evensHasher{
+		tracer:    e.tracer,
+		data:      make(chan flow.EventsList, len(collections)+1),
+		blockSpan: blockSpan,
+		callBack: func(hash flow.Identifier, err error) {
+			if err != nil {
+				panic(err)
+			}
+			res.EventsHashes = append(res.EventsHashes, hash)
+		},
+	}
+
 	go func() {
 		bc.Run()
 		wg.Done()
 	}()
 
+	go func() {
+		eh.Run()
```

Comment:
Does it make sense to make the `eventHasher` a `ReadyDoneAware` implementation? 
https://github.com/onflow/flow-go/blob/f861f317233ddec2dd49bf19f08928be0b2a9775/module/common.go#L3-L7